cogs/say.py

```diff
@@ -46,8 +46,10 @@ async def say(self, ctx: Context):
         if optionalarg is None:
             optionalarg = match.group('option2')
             if optionalarg is None:
-                optionalarg = (ctx.message.author.nick if ctx.message.author.nick
```

Comment:
This is from 1bb82cbb13ed1b6a7546719b917797807071c12c in master, but it's showing up here too because I rebased onto master which copies a bunch of commits over and reapplies the reddit branch changes over top.